scripts/list_boards.py

```diff
@@ -197,6 +197,15 @@ def find_v2_boards(args):
                         # Not the board we're looking for, ignore.
                         continue
 
+                board_revision = board.get('revision')
+                if(board_revision is not None and board_revision.get('format') != 'custom'):
```

Comment:
nit: Unnecessary parens
```suggestion
                if board_revision is not None and board_revision.get('format') != 'custom':
```